en/flight_controller/mro_control_zero_f7.md

```diff
@@ -0,0 +1,121 @@
+All accessed using front and rear 30pin Molex PicoClasp connectors.
+A durable plastic case, conformal board coating, and optional temperature calibration included.
+
+
+> **Note** This flight controller is [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+
+
+## Key Features
+
+- Microprocessor:
+  - 32-bit STM32F777 Cortex<sup>&reg;</sup> M4 core with FPU rev. 3
+  - 216 MHz/512 KB RAM/2 MB Flash
+  - F-RAM Cypress MF25V02-G 256-Kbit nonvolatile memory (Flash memory that performs as fast as RAM)
+- Sensors:
+  - [Bosch BMI088](https://www.bosch-sensortec.com/bst/products/all_products/bmi088_1) 3-axis accelerometer/gyroscope (internally vibration dampened)
+  - [Invensense ICM-20602](https://www.invensense.com/products/motion-tracking/6-axis/icm-20602/) 3-axis accelerometer/gyroscope
+  - [Invensense ICM-20948](https://www.invensense.com/products/motion-tracking/9-axis/icm-20948/) 3-axis accelerometer/gyroscope/magnetometer
+  - [Infineon DPS310 barometer](https://www.infineon.com/cms/en/product/sensor/barometric-pressure-sensor-for-consumer-applications/dps310/) (So smooth and NO more light sensitivity)
+
+- Interfaces:
+  - 6x UART (serial ports total), 3x with HW flow control, 1x FRSky Telemetry (D or X types), 1x Console and 1x GPS+I2C
+  - 8x PWM outputs (all DShot capable)
+  - 1x CAN
+  - 1x I2C
+  - 1x SPI
+  - Spektrum DSM / DSM2 / DSM-X® Satellite compatible input and binding
+  - Futaba S.BUS® & S.BUS2® compatible input
+  - FRSky Telemetry port output
+  - Graupner SUMD
+  - Yuneec ST24
+  - PPM sum input signal
+  - 1x JTAG (TC2030 Connector)
+  - 1x RSSI (PWM or voltage) input
+  - Tricolor LED
+
+- Weight and Dimensions (Uncased):
+  - Weight: 5.3g (0.19oz)
+  - Width: 20mm (0.79")
+  - Length: 32mm (1.26")
+
+- Power System:
+  - 3x Ultra low noise LDO voltage regulator
+
+
+
+## Purchase
+
+* [mRo Control Zero](https://store.mrobotics.io/mRo-Control-Zero-F7-p/mro-ctrl-zero-f7.htm)
+
+
+
+## Building Firmware
+
+> **Tip** Most users will not need to build this firmware!
+  It is pre-built and automatically installed by *QGroundControl* when appropriate hardware is connected.
+
+To [build PX4](https://dev.px4.io/master/en/setup/building_px4.html) for this target:
+```
+make mro_ctrl-zero-f7
+```
+
+
+## Debug Ports
+
+### Console Port
+
+The [PX4 System Console](https://dev.px4.io/master/en/debug/system_console.html) runs on `USART7` using the pins listed below.
+This is a standard serial pinout, designed to connect to a [3.3V FTDI](https://www.digikey.com/product-detail/en/TTL-232R-3V3/768-1015-ND/1836393) cable (5V tolerant).
+
+mRo control zero f7 | | FTDI | 
+--- | --- | --- | ---
+17 | USART7 Tx   | 5 | FTDI RX (yellow)
+19 | USART7 Rx   | 4 | FTDI TX (orange)
+6  | USART21 GND | 1 | FTDI GND (black)
+
+
+### SWD Port
+
+The [SWD port](http://dev.px4.io/master/en/debug/swd_debug.html) (JTAG) for FMU debugging is shown below.
+
+<!-- insert image here -->
```

Comment:
FYI, I can create image when I know which port it is and how we connect to it - plug, solder, whatever (if solder, need pinout if does not match the connector). This is discussed inhttps://github.com/PX4/px4_user_guide/pull/716/files#r411092655

Reply:
@hamishwillee The Tag connect plugs into the board See https://www.tag-connect.com/. It is held in place by [swedge  pins](https://www.digikey.com/product-detail/en/microchip-technology/TC2030-CLIP-3PACK/TC2030-CLIP-3PACK-ND/3597496). There is a [TC to 20 pin Jlink](https://www.tag-connect.com/product/arm20-ctx-20-pin-to-tc2030-idc-adapter-for-cortex)

Reply:
Thanks! That is very helpful.